runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/RunecraftAction.java

```diff
@@ -75,10 +77,34 @@ public enum RunecraftAction implements ItemSkillAction
 	private final float xp;
 	private final boolean ignoreBonus;
 	private final boolean isMembersOverride;
+	private final String nameOverride;
+
+	@Override
+	public String getName(final ItemManager itemManager)
```

Comment:
Instead of overriding `getName()` for all runecraft actions you can just override it on `TRUE_BLOOD_RUNE` itself, similar to https://github.com/LootBagger/runelite/blob/015798209471f9d62914035f3987e9ffdb5c5eaf/runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/SmithingAction.java#L120-L133

Reply:
I see. Yeah that will be more clean.